Workbooks/WorkbooksMetadata.json

```diff
@@ -970,5 +970,18 @@
   "templateRelativePath": "MITREAttack.json",
   "subtitle": "",
   "provider": "Azure Sentinel community"
+  },
+  {
+  "workbookKey": "InvestigationInsightsWorkbook",
+  "logoFileName": "Microsoft_logo.svg",
+  "description": "Help analysts gain insight into Incident and Entity data through the Investigation Insights Workbook.  This workbook provides common queries and detailed visualizations to help an analyst investigate suspicious activities quickly with an easy to use interface. Analysts can start their investigation from a Sentinel Incident, or by simply entering the entity data into the workbook manually.",
+  "dataTypesDependencies": [ "AuditLogs", "AzureActivity", "CommonSecurityLog", "OfficeActivity", "SecurityEvent", "SigninLogs", "ThreatIntelligenceIndicator" ],
+  "dataConnectorsDependencies": [ "AzureActivity", "SecurityEvents", "Office365", "AzureActiveDirectory" ],
+  "previewImagesFileNames": [ "InvestigationInsightsWhite1.png", "InvestigationInsightsBlack1.png", "InvestigationInsightsWhite2.png", "InvestigationInsightsBlack2.png" ],
+  "version": "1.0",
+  "title": "Investigation Insights",
+  "templateRelativePath": "InvestigationInsights.json",
+  "subtitle": "",
+  "provider": "Microsoft"
```

Comment:
please change the provider to "Azure Sentinel community" to avoid support dependency on the Azure Sentinel engineering team

Reply:
updated